packages/cli/package.json

```diff
@@ -24,18 +23,19 @@
     "lodash": "4.17.21",
     "ora": "2.1.0",
     "path-exists": "3.0.0",
-    "sanitize-filename": "1.6.3",
```

Comment:
`sanitize-filename` is not in use anymore?

Reply:
@JosefBredereck exactly. I‘ce just recognized that I seem to have its removal checked in with this code whether I actually wanted to open an distinct pull request for its removal to keep the things seperated. I‘ll have a look at this tomorrow and will provide further information as well.